app/controllers/routes_controller.rb

```diff
@@ -7,11 +7,15 @@ def routes
   end
 
   def pass_checkpoint
-    Route.find(routes_params[:ids]).update(is_passed: true, pass_date: routes_params[:pass_date])
+    route = Route.find(routes_params[:ids])
+    route.update(is_passed: true, pass_date: routes_params[:pass_date])
+    render route
```

Comment:
Maybe something is missing here, am not sure. but i get this error when trying to pass checkpoint as driver
![image](https://user-images.githubusercontent.com/70818761/163280682-55cf4a32-e511-4857-98e2-3876e26aba60.png)